frontend/nginx.conf

```diff
@@ -0,0 +1,26 @@
+user nginx;
+
+worker_processes 1;
+
+events { worker_connections 1024; }
+
+http {
+    include       /etc/nginx/mime.types;
+    server {
+
+        listen 80;
```

Comment:
Can we make this port value configurable via environment variable, or if not, make it 8000? On ECS everything is running on 8000 and that port is opened in the security group rules as well. Running something else on port 80 would break this uniformity (we need to open an other port + remember that frontend is not 8000).

Reply:
Changed nginx port to 8000